src/Apps/Auction/Components/AuctionActiveBids.tsx

```diff
@@ -237,10 +249,24 @@ const BidButton: React.FC<
     )
   }
 
+  if (!!sale.isLiveOpen) {
+    return (
+      <Button
+        // @ts-ignore
+        as={RouterLink}
+        to={`${getENV("PREDICTION_URL")}/${sale.slug}/login`}
+        width="50%"
+        variant="primaryBlack"
+      >
+        Enter live bidding
+      </Button>
+    )
+  }
+
   // By default, we redirect to /artwork/id on successful bid, but since we're
   // in the ActiveBids component, redirect to /auction/id (i.e., close modal)
-  const redirectTo = `/auction/${lotStanding.saleArtwork?.saleID}`
-  const href = `/auction/${lotStanding.saleArtwork?.saleID}/bid/${lotStanding?.saleArtwork?.slug}?redirectTo=${redirectTo}`
+  const redirectTo = `/auction/${saleArtwork.saleID}`
+  const href = `/auction/${saleArtwork.saleID}/bid/${saleArtwork.slug}?redirectTo=${redirectTo}`
```

Comment:
non-blocking: might be nice to move all these url constructors including the getENV one above into simple descriptively-named functions outside the component.

Reply:
This whole thing can use some major refactors. I did not change anything about this except of storing saleArtwork into a variable. If we end up working more in this area I think we should do bunch of consolidatinf between here and settings/auctions and MyBids on the auction page. For now not messing with any of this.